### PR TITLE
Tests for recent commits and Robustness against non-JSON-formatted delay tasks (4-2-stable)

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

File: packaging/test_plugin_indexing.py [hunk 1]
```diff
@@ -240,11 +271,13 @@ def remove_all_jobs_from_delay_queue(self, session_, initial_sleep=0.0):
         session_.assert_icommand('iqdel -a', 'STDERR')
         self.fail_unless_delay_queue_is_empty( session_ )
 
-    def delay_queue_is_empty(self, session_, soft_assert = False, try_kill = False):
+    def delay_queue_is_empty(self, session_, soft_assert = False, try_kill = False, Verbose = False):
```
Comment (trel) [inline packaging/test_plugin_indexing.py:274]: lowercase Verbose?

**Comment by trel** [comment]
a few places with shortened variable names, but they were readable.     what else is left for this PR to do?

**Comment by d-w-moore** [comment]
This is it. Giving it a good check-over, and it needs to be tested in CI but now working at bench.

**Comment by d-w-moore** [comment]
passing in Jenkins CI

**Comment by trel** [comment]
one left - but willing to ignore if you are.     if ready, please squash to make clean - we'll # after that.

**Comment by d-w-moore** [comment]
Yes, will push the same change to master and # both. It's all ready.

**Pull request merged**